src/app/file-manager/file-explorer/file-explorer.component.ts

```diff
@@ -18,23 +17,18 @@ import { FilesDataSource } from '@app/file-manager/dbstate/files-datasource';
   styleUrls: ['./file-explorer.component.scss']
 })
 export class FileExplorerComponent implements OnInit {
+  IS_HOME: Boolean = false;
```

Comment:
If this isn't a constant it should probably be named like a regular field, e.g. `isHome`